Assets/Scripts/UI/InGame/Scoreboard/Scoreboard.cs

```diff
@@ -0,0 +1,195 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+using TMPro;
+
+namespace Assets.Scripts.UI.InGame.Scoreboard
+{
+    public class Scoreboard : UiMenu
+    {
+        public GameObject playerInfoPrefab;
+        public GameObject playerListParent;
+
+        public SortButton[] sortButtons;
+
+        public TMP_Text roomName;
+        public TMP_Text playerCount;
+        public TMP_Text gameMode;
+        public TMP_Text sortContainer;
+        public string sortLabel;
+        private float timeLast = 0;
+
+
+        protected override void OnEnable()
+        {
+            base.OnEnable();
+        }
+
+        protected override void OnDisable()
+        {
+            base.OnDisable();
+        }
+
+
+        void Start()
+        {
+            sortButtons = GameObject.FindObjectsOfType<SortButton>();
+        }
+
+        private void LateUpdate()
+        {
+            // UpdateScoreboard should be called 8 times a second. 
```

Comment:
So, rather make that a property. "UpdatePerSecond". Interval = 1000 / UpdatePerSecond